vantage6-client/vantage6/client/__init__.py

```diff
@@ -1847,6 +1851,11 @@ def create(
                 at least a 'label' key. Additional keys are 'query' (if using
                 SQL/SPARQL databases), 'sheet_name' (if using Excel databases),
                 and 'preprocessing' information.
+            action: AlgorithmStepType, optional
+                Session action type to be performed by the task. If not provided, the
+                action from the algorithm store will be used. Suitable actions may be
```

Comment:
This is not guaranteed to exist right?
```suggestion
                action from the algorithm store will be used if available. Suitable actions may be
```

Reply:
Yes, I updated the docstring with clear information (I did not accept your suggestion because of line length stuff)